bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml

```diff
@@ -21,11 +21,11 @@ metadata:
     containerImage: quay.io/opentelemetry/opentelemetry-operator
     createdAt: "2020-12-16T13:37:00+00:00"
     description: Provides the OpenTelemetry components, including the Collector
-    operators.operatorframework.io/builder: operator-sdk-v1.2.0
```

Comment:
As mentioned in the comments, please revert this file.

Reply:
@jpkrohling TFTR!
reverted the files